modules/persistent-storage-csi-azure-disk-perf-plus-limits.adoc

```diff
@@ -0,0 +1,19 @@
+//
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-azure.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-azure-disk-perf-plus-limits_{context}"]
+= Limitations
+
+Performance plus for Azure Disk has the following limitations:
+
+* Can be enabled only on Standard HDD, Standard SSD, and Premium SSD managed disks that are 513 GiB or larger.
+
+* Can be enabled only on new disks. 
++
+*Workaround:* Create a snapshot of a disk that has performance plus enabled on it, and then create a new disk from the snapshot.
+
+* Not supported for disks recovered with Azure Site Recovery.
```

Comment:
Hmm, should we mention this? I don't think we need to cover "Azure Site Recovery".

Reply:
yeh we can remove it, this can introduce confusion

Reply:
Done.